sandbox/public/onBeforeEventSentHelper.js

```diff
@@ -0,0 +1,40 @@
+// code to collect tnta and trigger the Analytics Hit
+const trackingServer = "ujsl.sc.omtrdc.net";
+const reportSuite = "ujslecommerce";
+
+function triggerAnalyticsHit({ analyticsPayload, visitorID }) {
+  const url = `https://${trackingServer}/b/ss/${reportSuite}/0?g=${window.location}&r=${document.referrer}&mid=${visitorID}&tnta=${analyticsPayload}`;
+
+  return fetch(url)
+    .then(success => {
+      console.log("success", success);
+    })
+    .catch(error => {
```

Comment:
Should this trigger and fail the whole alloy request as well?

Reply:
nope, we shouldn't change any alloy requests, just retrieve the information needed from the event.